cli/src/main/resources/application.yaml

```diff
@@ -41,6 +41,12 @@ quarkus:
     console:
       format: "%d{HH:mm:ss,SSS} %-5p [%c{3.}] (%t) %s%e mdc:[%X]%n"
       stderr: true
+  oidc-client:
+    enabled: true
+    auth-server-url: http://localhost:8180/auth/realms/quarkus
```

Comment:
@jomrazek This needs to be configured to be our internal SSO right? We have private files for this, but double checking it needs to be configured. Thanks!

Reply:
Yes on internal config files this will be configured to RedHat internal SSO system